attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -24,18 +38,32 @@ struct TdxEvidence {
     quote: String,
 }
 
+pub fn make_attester() -> Box<dyn Attester + Sync + Send> {
+    if az::detect_platform() {
+        Box::<az::TdxAttester>::default()
+    } else {
+        Box::<TdxAttester>::default()
+    }
+}
+
+fn convert_report_data(report_data: String) -> Result<Vec<u8>> {
+    let mut report_data_bin = base64::decode(report_data)?;
+    if report_data_bin.len() != 48 {
+        return Err(anyhow!(
+            "TDX Attester: Report data should be SHA384 base64 String"
+        ));
```

Comment:
`anyhow::bail!` macro helps to make it shorter
```suggestion
        bail!("TDX Attester: Report data should be SHA384 base64 String");
```

Reply:
i didn't know this, but i like it 👍